website/blog/2025-09-25-hacktoberfest.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Hacktoberfest 2025! - Contribute to Podman Desktop
+description: Contribute to Podman Desktop this Hacktoberfest!
+slug: hacktoberfest-2025
+authors: [rujutashinde]
+tags: [podman-desktop, hacktoberfest, community]
+hide_table_of_contents: false
+---
+
+# Hacktoberfest 2025! - Contribute to Podman Desktop
+
+October is here and [**Hacktoberfest**](https://hacktoberfest.com/) is back! It’s the perfect opportunity to sharpen your skills, explore open source, and contribute to [**Podman Desktop** ](https://podman-desktop.io/) — an open source tool designed to simplify working with containers and Kubernetes from your local machine.
+
+## Why Contribute to Podman Desktop?
+
+Podman Desktop is a [CNCF](https://www.cncf.io/) sandbox project and has already grown into a thriving open source community with over [**3 million downloads worldwide**](https://podman-desktop.io/blog/3-million). By contributing, you’re not only helping shape a successful, widely used project, but also joining a community of developers building tools that benefit peers and the broader ecosystem.
+
+Podman Desktop is built with developers in mind. **Designed by developers, for developers** !!
+It’s a cross-platform tool that helps streamline container workflows, offering features like:
+
+- **Container management** Manage your containers, images, and pods directly from your desktop.
+- **Kubernetes integration** Run and test workloads in a local Kubernetes environment.
+- **Extension framework** Build custom extensions to expand and personalize the desktop experience.
+- **Modern tech stack** Contribute using TypeScript, Svelte, Tailwind CSS, Electron, Playwright and more!
+
+Whether you’re fixing a bug, enhancing the UI, or creating a new extension, your contributions will help shape the next generation of container tooling.
+
+## Contribution Areas
+
+Here are some ways developers can get involved:
+
+- 🐛 **Bug Fixes** – Help identify and resolve issues in the desktop app.
```

Comment:
I feel we should link each of these bullet points with links to Issues section, Documentation, Extensions page, etc.

Reply:
done